### PR TITLE
Content length

### PR DESCRIPTION
**Pull request by gregwebs**
Add Network.Wai.Header.contentLength helper for reading the Content-Length header

### CODE REVIEW

File: wai-extra/Network/Wai/Middleware/RequestLogger.hs [hunk 1]
```diff
@@ -308,10 +305,7 @@ detailedMiddleware' cb ansiColor ansiMethod ansiStatusCode app req sendResponse
     collectPostParams (postParams, files) = postParams ++
       map (\(k,v) -> (k, "FILE: " <> fileName v)) files
 
-    readInt bs =
```
Comment (gregwebs) [inline wai-extra/Network/Wai/Middleware/RequestLogger.hs]: This is actually different than S8.readInteger. I was assuming that was an accident, but maybe there is a purpose?

**Comment by snoyberg** [comment]
Isn't this already handled by [requestBodyLength](http://haddock.stackage.org/lts-3.9/wai-3.0.4.0/Network-Wai.html#v:requestBodyLength)?

**Comment by gregwebs** [comment]
one is for the request and the other for the response. Thanks for the reminder about requestBodyLength, I will have it use that

**Comment by gregwebs** [comment]
closing in favor of #453

**Pull request closed without merging**